Extensions/timestamps.js

```diff
@@ -245,14 +363,15 @@ XKit.extensions.timestamps = new Object({
 		if (!cached_date.isValid()) {
 			return false;
 		}
+
 		date_element.html(this.format_date(cached_date));
 		date_element.removeClass("xtimestamp_loading");
 		return true;
 	},
 
 	show_failed: function(obj) {
 		$(obj).html("failed to load timestamp <div class=\"xkit-timestamp-failed-why\">why?</div>");
-		$(obj).removeClass('xtimestamp_loading');
+		if ($(obj).hasClass('xtimestamp_loading')) { $(obj).removeClass('xtimestamp_loading'); }
```

Comment:
redundant; `.removeClass` already tests for classes before removing them, or otherwise does nothing if the class to be removed isn't there

```suggestion
		$(obj).removeClass('xtimestamp_loading');
```